src/Files.App/ViewModels/UserControls/NavigationToolbarViewModel.cs

```diff
@@ -1200,49 +1148,41 @@ void AddNoResultsItem()
 
 		public void PopulateOmnibarSuggestionsForCommandPaletteMode()
 		{
-			OmnibarCommandPaletteModeText ??= string.Empty;
-			OmnibarCommandPaletteModeSuggestionItems.Clear();
-
 			if (ContentPageContext.SelectedItems.Count == 1 && ContentPageContext.SelectedItem is not null && !ContentPageContext.SelectedItem.IsFolder)
 			{
-				var dispatcherQueue = Microsoft.UI.Dispatching.DispatcherQueue.GetForCurrentThread();
-
-				dispatcherQueue.TryEnqueue(() =>
```

Comment:
I found that `DispatcherQueue` is required for actions to work properly. Unfortunately, we can't do any further testing in this area right now because the experimental API doesn't seem to work in the latest Insider build.

Reply:
I see, thanks.

Reply:
It looks like it's missing from the `Omnibar_TextChanged` event. To prevent this from being missed in the future, it would be safer to revert this change.